libraries/WiFi/examples/WiFiClient/WiFiClient.ino

```diff
@@ -15,9 +15,13 @@
  */
 
 #include <WiFi.h>
+#include "secrets.h"
```

Comment:
If we're going to be relying on secrets.h this heavily, there should be a short comment above the #include line explaining what's going on in that file. Then, link to `secrets.rst` or any related documentation that explains more than a sentence.

Reply:
Good idea, thank you.